libs/core/src/lib/quick-view/quick-view-group-item-label/quick-view-group-item-label.component.ts

```diff
@@ -1,8 +1,25 @@
-import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { AfterViewInit, ChangeDetectionStrategy, Component, ElementRef } from '@angular/core';
 
 @Component({
     selector: 'fd-quick-view-group-item-label',
     templateUrl: './quick-view-group-item-label.component.html',
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class QuickViewGroupItemLabelComponent {}
+export class QuickViewGroupItemLabelComponent implements AfterViewInit {
+    /** @hidden */
+    constructor(private readonly _elRef: ElementRef) {}
+
+    /** @hidden */
+    ngAfterViewInit(): void {
+        this._bindElementAttributes();
+    }
+
+    /** @hidden */
+    private _bindElementAttributes(): void {
```

Comment:
Could you please choose another name for the method? Or provide some description to the method so in future people would know what exactly it's doing. Thank you!